moveit_commander/CMakeLists.txt

```diff
@@ -10,4 +10,9 @@ catkin_package()
 install(PROGRAMS bin/${PROJECT_NAME}_cmdline.py
         DESTINATION ${CATKIN_PACKAGE_BIN_DESTINATION})
 
-add_subdirectory(test)
+if(CATKIN_ENABLE_TESTING)
```

Comment:
What's the benefit of merging the two cmake files?
Here, the difference is not so important, but usually you should use a one cmake file per directory:
https://pabloariasal.github.io/2018/02/19/its-time-to-do-cmake-right/

Reply:
I have drop this in the new verion